Source/ACE/Network/GameMessages/Messages/GameMessageCreateLifestone.cs

```diff
@@ -31,7 +31,7 @@ public GameMessageCreateLifestone(Player player) : base(GameMessageOpcode.Object
             Position newPosition = new Position(player.Position.Cell, player.Position.Offset.X + dx, player.Position.Offset.Y + dy, player.Position.Offset.Z + 0.5f, 0f, 0f, 0f, 0f);
             
             var weenie = WeenieHeaderFlag.Usable | WeenieHeaderFlag.BlipColour | WeenieHeaderFlag.Radar | WeenieHeaderFlag.UseRadius;
-            ImmutableWorldObject wo = new ImmutableWorldObject(ObjectType.LifeStone, new ObjectGuid(nextObjectId++, GuidType.None), "Life Stone", ObjectDescriptionFlag.LifeStone, weenie, newPosition);
+            ImmutableWorldObject wo = new ImmutableWorldObject(ObjectType.LifeStone, new ObjectGuid(GuidManager.NewStaticObjectGuid()), "Life Stone", 1, ObjectDescriptionFlag.LifeStone, weenie, newPosition);
```

Comment:
I am not sure how to fix it, but the way its in ace now its not named correctly.
Also, passing a 1, its required now.. now sure the proper way to use this.. but interested..
can we keep this now until its in properly ?